code/src/index.css

```diff
@@ -1,13 +1,61 @@
+
 body {
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+
+.todo-container {
+  width: 100%;
+  min-height: 400px;
+
+}
+
+.todo-header {
+  background-color: silver;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  border-bottom: solid grey 1px;
+  padding: 50px;
+  
+
+}
+
+.add-todo {
+  width: 60%;
+  padding: 20px;
+}
+
+.add-todo-button {
+  width: 28%;
+  height: 59px;
+}
```

Comment:
60% + 28% =/= 100% so that is why your form is not the same width as the rest of your project.

Reply:
Hey Karol, I've changed everything you commented on, and also changed the styling. https://malins-todo-list.netlify.app/